src/test/views/test_donaletty.py

```diff
@@ -34,88 +14,29 @@ def ptz_data(inputdatapath, outputdatapath):
         latitude,
         longitude,
         scanid,
-        ecmwfpath=os.path.join(inputdatapath, "ERA-Interim"),
-        solardatafile=os.path.join(inputdatapath, "Solardata2.db"),
-        zptpath=outputdatapath,
-    )
-    assert os.path.isfile(
-        os.path.join(outputdatapath, "2015", "01", "ZPT_{}.nc".format(scanid))
     )
+    assert data
     return data
 
 
-@pytest.mark.parametrize(
-    "basedir,current_date,hour,expect",
-    (
-        ("/test1", date(2019, 8, 31), "12", "/test1/2019/08/ei_pl_2019-08-31-12.nc"),
-        ("/test2", date(2019, 8, 31), "18", "/test2/2019/08/ei_pl_2019-08-31-18.nc"),
-        ("/test3", date(2019, 9, 1), "00", "/test3/2019/09/ea_pl_2019-09-01-00.nc"),
-        ("/test4", date(2019, 9, 1), "06", "/test4/2019/09/ea_pl_2019-09-01-06.nc"),
-    ),
-)
-def test_get_filepath(basedir, current_date, hour, expect):
-    donaletty = Donaletty(current_date, "dummy", basedir)
-    filename = donaletty.get_filepath(current_date, hour)
-    assert filename == expect
-
-
-def test_loadecmwfdata_reads_expected_files(inputdatapath):
-    basedir = os.path.join(inputdatapath, "ERA-Interim")
-    current_datetime = datetime(2015, 1, 12, 1)
-    donaletty = Donaletty(current_datetime, "dummy", basedir)
-    donaletty.loadecmwfdata()
-    assert [donaletty.ecm[0]["fid"].filepath(), donaletty.ecm[1]["fid"].filepath()] == [
-        os.path.join(basedir, "2015/01/ei_pl_2015-01-12-00.nc"),
-        os.path.join(basedir, "2015/01/ei_pl_2015-01-12-06.nc"),
-    ]
-
-
-@pytest.mark.parametrize(
-    "parameter,index,expect",
-    (
-        ("z", 0, 75.0),
-        ("z", 75, 150.0),
-        ("p", 0, 0.02448),
-        ("p", 75, 4.2824e-6),
-        ("t", 0, 212.93),
-        ("t", 75, 706.19),
-    ),
-)
-def test_msis90_values(inputdatapath, parameter, index, expect):
-    msis90 = Msis90(solardatafile=os.path.join(inputdatapath, "Solardata2.db"))
-    latitude = 11.1
-    longitude = 82.0
-    altitudes = np.arange(75, 151, 1)
-    p, t, z = msis90.extractPTZprofilevarsolar(
-        datetime(2015, 1, 12, 1), latitude, longitude, altitudes
-    )
-    data = {"p": p, "t": t, "z": z}
-    assert data[parameter].shape == (76,) and data[parameter][index] == pytest.approx(
-        expect, rel=1e-3
-    )
-
-
-@pytest.mark.slow
 def test_donaletty_returns_expected_parameters(ptz_data):
     assert set(ptz_data) == set(
-        ["ScanID", "Z", "P", "T", "latitude", "longitude", "datetime"]
+        ["ScanID", "Z", "P", "T", "latitude", "longitude", "mjd"]
     )
 
 
-@pytest.mark.slow
 @pytest.mark.parametrize(
     "parameter,expect",
     (
         ("ScanID", 7014821387),
-        ("latitude", 11.1),
-        ("longitude", 82.0),
+        ("latitude", 11.66165),
+        ("longitude", 82.395362),
```

Comment:
What happened here?

Reply:
same thing as with the apriori file, now the source is from the "live data", not from the test-data set.